lib/sucker_punch/job.rb

```diff
@@ -21,8 +21,10 @@ module Job
     def self.included(base)
       base.extend(ClassMethods)
       base.class_attribute :num_workers
+      base.class_attribute :max_jobs
```

Comment:
Because the `class_attribute` is internal and we have to create a class method for use in the Job as something different, let's keep the pattern consistent with variables:

```ruby
base.class_attribute :num_jobs_max
```
That'll require it get updated everywhere else.